src/rgw/rgw_policy_s3.cc

```diff
@@ -276,6 +276,10 @@ int RGWPolicy::from_json(bufferlist& bl, string& err_msg)
       int i;
       for (i = 0; !citer.end() && i < 3; ++citer, ++i) {
 	JSONObj *o = *citer;
+        string condition_data = o->get_data();
```

Comment:
better off using a const reference for holding the returned value of `o->get_data()`. and reference it at line https://github.com/ceph/ceph/pull/11572/commits/e887f62ff8f9602d1ee785f5e256079296554112#diff-d71c299d7f07bbeccbb5a0d3aca46659R283